### ISSUE TITLE
JSON formatter produces invalid JSON file

### ISSUE THREAD

**Issue by dsiguero** [robusta-dev/krr#202]
**Describe the bug**
`krr simple --logtostderr -f json > result.json` does not produce valid JSON.

**To Reproduce**
```
$ krr simple --logtostderr -f json > result.json
[...]

$ jq '.' result.json
jq: parse error: Invalid numeric literal at line 1, column 12

$ head -n1 result.json
Calculating Recommendation |████████████████████████████████████████| 50 in 10.8s (4.61/s)
```

**Expected behavior**
The resulting JSON file should be valid.

**Are you interested in contributing a fix for this?**
Yes.

**Additional context:**
 - OS: macOS Sonoma 14.2.1
 - Installation method: `brew`
 - `krr` version: `v1.6.0`

**Comment by dsiguero** [comment]
This seems to also apply to the YAML formatter as well, when running the following command:

```
$ krr simple --logtostderr -f yaml > result.yaml
[...]
$ yq '.' result.yaml
Error: bad file 'out.yaml': yaml: line 2: mapping values are not allowed in this context
$ head -n1 result.yaml
Calculating Recommendation |████████████████████████████████████████| 50 in 11.2s (4.46/s)
```

**Comment by rohank07** [comment]
I believe this is resolved on the main branch - waiting on a new release

**Comment by aantn** [comment]
Hey all, I believe this is fixed on recent releases so I'm closing. If you experience any more issues please reopen or report a new issue. 

Thank you for using KRR!
